reporting_app/static/column_formatter.js

```diff
@@ -156,6 +185,32 @@ function count_entities_fmt(data, fmt){
     return data.length;
 }
 
+function temporal_fmt(cell_data, fmt){
+/*
+ * Returns formatting style is for project status page, displaying a green, yellow or red if it is over a week,
+ * two weeks or four weeks since the last change.
+ */
+    // Checking time elapsed of the status' max date
+    status_date = new Date(cell_data['last_modified_date']);
+    // Creating fixed date variable to compare against
+    week_ago = new Date();
+    week_ago.setDate(week_ago.getDate() - 7)
+    two_weeks_ago = new Date();
+    two_weeks_ago.setDate(week_ago.getDate() - 14)
+    four_weeks_ago = new Date();
+    four_weeks_ago.setDate(week_ago.getDate() - 28)
```

Comment:
This section could be simplify using moment.js 
```javascript
    days_since_last_change = moment().diff(cell_data['last_modified_date'], 'days')
```
The compare `days_since_last_change` with the thredholds

Reply:
Do we have moment.js installed?

Reply:
Great tip, worked like a charm. Thanks.